config/application.docker.yml

```diff
@@ -0,0 +1,64 @@
+# Add application configuration variables here, as shown below.
```

Comment:
Why not use the example file?

Reply:
Good idea! It will reduce `docker` footprint.

Reply:
Are there any variables that need to be customised?